admin/packaging.py

```diff
@@ -864,7 +880,23 @@ def omnibus_package_builder(
             BuildPackage(
                 package_type=distribution.package_type(),
                 destination_path=destination_path,
-                source_paths={flocker_node_path: FilePath("/usr/sbin")},
+                source_paths={
+                    flocker_node_path: FilePath("/usr/sbin"),
+                    # Fedora/CentOS firewall configuration
+                    package_files.child('flocker-control-api.firewalld.xml'):
```

Comment:
At least firewalld is available on [ubuntu](http://packages.ubuntu.com/search?keywords=firewalld). So, it seems to make sense to install the configuration files for it, even if it isn't used by default.